node_scripts/docker_main.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 # This file is the entry point of the docker container.
-# It will run the custom scripts if present and start spark.
+# It will setup WASB and start Spark.
```

Comment:
worth commenting that it currently uses the same storage account as the one configured in the secrets.yaml (or conversely that the one in secrets.yaml is used for moving data around for this tool to work)